toolchain/driver/runtimes_cache_test.cpp

```diff
@@ -640,8 +642,8 @@ TEST_F(RuntimesCacheTest, LookupWithManyStaleRuntimes) {
   EXPECT_THAT(runtimes2.base_dir().Stat()->unix_inode(), Eq(runtimes2_inode));
 
   // One of the stale runtimes should be freshly created though.
-  EXPECT_THAT(stale_runtimes_0.base_dir().Stat()->unix_inode(),
-              Ne(stale_runtimes_0_inode));
+  EXPECT_THAT(stale_runtimes_0.base_dir().Stat()->mtime(),
+              Ne(stale_runtimes_0_mtime));
```

Comment:
Seems even more reliable to check both?
```suggestion
  EXPECT_FALSE(stale_runtimes_0.base_dir().Stat()->unix_inode()
               == stale_runtimes_0_inode &&
               stale_runtimes_0.base_dir().Stat()->mtime()
               == stale_runtimes_0_mtime);
```

Reply:
Right. I also forgot to consider that on older filesystems, mtime might not be so precise, so this might also be flaky on something like ext3 (not that anyone is using that here). This is probably _still_ flaky, but the chances are much much lower.